.babelrc

```diff
@@ -1,5 +1,5 @@
 {
-  "presets": ["es2015", "react", "stage-1"],
+  "presets": ["latest", "react", "stage-1"],
```

Comment:
Could you please make this change in a separate PR against the master branch? I already wanted to replace `babel-preset-es2015` for that. :)